sources/packages/backend/apps/db-migrations/src/migrations/1724973005363-AddAnnouncements.ts

```diff
@@ -0,0 +1,19 @@
+import { MigrationInterface, QueryRunner } from "typeorm";
+import { getSQLFileData } from "../utilities/sqlLoader";
+
+export class AddAnnouncements1724973005363 implements MigrationInterface {
```

Comment:
CreateAnnouncements ?

Reply:
'Add' seemed to be the more common name in previous migrations.

Reply:
It is not a blocker. Resolving.

Reply:
"Create" is good for me.

Reply:
I see "Create" being used for table creation and "Add" being used also many times for other additions.
Please do not extend this conversation. It seems that "Create" would be the best fit but it would not be a blocker IMO.

Reply:
updated